deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-fpga-plugin
+  namespace: kube-system
+  labels:
+    app: intel-fpga-plugin
+spec:
+  selector:
+    matchLabels:
+      app: intel-fpga-plugin
+  template:
+    metadata:
+      labels:
+        app: intel-fpga-plugin
+    spec:
+      serviceAccountName: intel-fpga-plugin-controller
+      initContainers:
+      - name: intel-fpga-initcontainer
+        image: intel/intel-fpga-initcontainer:devel
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          readOnlyRootFilesystem: true
+        volumeMounts:
+        - mountPath: /opt/intel/fpga-sw
+          name: intel-fpga-sw
+        - mountPath: /etc/containers/oci/hooks.d
+          name: oci-hooks-config
+      containers:
+      - name: intel-fpga-plugin
+        env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        image: intel/intel-fpga-plugin:devel
+        imagePullPolicy: IfNotPresent
+        command:
+          - /usr/local/bin/intel_fpga_device_plugin
```

Comment:
we dropped the `command` earlier (the image sets `ENTRYPOINT`). Did you hit a problem?

Reply:
The admission webhook image sets `ENTRYPOINT`, but `intel-fpga-plugin.Dockerfile` sets `CMD`. Therefore setting only `args:` without `command:` did not work.

@mythi , do you think we could change to `ENTRYPOINT` also in this image - and do so in this patch? Or do you think that we should do this in more careful manner not to break anything?

Reply:
It looks you hit [an undefined behavior](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes).

I'd suggest we set the image `ENTRYPOINT` in this PR.

Reply:
`ENTRYPOINT` added.